integration/terraform/ec2/win/vpc.tf

```diff
@@ -20,9 +21,9 @@ resource "aws_security_group" "ec2_security_group" {
   }
 
   ingress {
-    from_port   = "22"
-    to_port     = "22"
-    protocol    = "tcp"
+    from_port   = 0
+    to_port     = 0
+    protocol    = "-1"
```

Comment:
so this is completely open now?

Reply:
It should not be. OpenSSH should opens port 22 for this.